misc/patternfly-showcase.css

```diff
@@ -0,0 +1,537 @@
+.text-overflow-pf {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  word-wrap: normal;
+}
+header + .container-fluid {
+  padding-bottom: 50px;
+  padding-top: 20px;
+  transition: padding-left 0.2s cubic-bezier(0.35, 0, 0.25, 1);
+}
+header .navbar.navbar-default {
```

Comment:
Hi, this will affect every ```<header>``` element.  We need to just limit this to ```h3```'s under ```showcase```.  Ie:
```
.showcase > header {
   ...
}
```